libs/application/template-api-modules/src/lib/modules/shared/api/user-profile/user-profile.service.ts

```diff
@@ -67,4 +91,8 @@ export class UserProfileService extends BaseTemplateApiService {
         throw error
       })
   }
+
+  private getIDSLink(application: ApplicationWithAttachments) {
+    return `${this.idsClientConfig.issuer}/app/user-profile/email?state=update&returnUrl=${this.idsClientConfig.redirectUri}/ellilifeyrir/${application.id}`
```

Comment:
Should get the slug from the application type. This is a shared provider so it should work for every application.

Reply:
ofc 🙈 will fix this